### ISSUE TITLE
Finish Processing | Show correct steps based on election option

### ISSUE THREAD

**Issue by shanear** [department-of-veterans-affairs/caseflow#3437]
As a claims assistant, I want to see the correct options to finish processing the RAMP election letter. These options are based on the election option selected on the Review request page.

## Acceptance Criteria
- If the election option selected on the "Review request" page was "Supplemental Claim", show the Supplemental Claim copy (see below)
- If the election option selected on the "Review request" page was "Higher level review with informal conference" or "Higher level review", show the Higher-Level Review copy (see below).
- An acknowledgment with a checkbox displays below the issues on appeal. It is unselected by default. It should operate according to the styleguide.
- Note: development for issues section shown in mock is covered in #3583 
- Withdrawn should no longer be an option

### Supplemental Claim
Header: Finish processing Supplemental Claim election
Subheader: Please complete the following steps outside Caseflow.

<b>Step 1.</b> Upload the RAMP Election form to the VBMS eFolder and ensure the Document Type is <b>Correspondence.</b>
<b>Step 2.</b> Update the Subject Line with "Ramp Election".
<b>Step 3.</b> Create an <b>EP 683 RAMP – Supplemental Claim Review Rating</b> in VBMS.
<b>Step 4.</b> Add a placeholder contention of "RAMP".
<b>Step 5.</b> Send a <b>RAMP Withdrawal Letter</b> using <i>Letter Creator.</i>

### Higher-Level Review
Header: Finish processing Higher-Level Review election
Subheader: Please complete the following steps outside Caseflow.

<b>Step 1.</b> Upload the RAMP Election form to the VBMS eFolder and ensure the Document Type is <b>Correspondence.</b>
<b>Step 2.</b> Update the Subject Line with "Ramp Election".
<b>Step 3.</b> Create an <b>EP 682 RAMP – Higher Level Review Rating</b> in VBMS.
<b>Step 4.</b> Add a placeholder contention of "RAMP".
<b>Step 5.</b> Send a <b>RAMP Withdrawal Letter</b> using <i>Letter Creator.</i>

## Mocks
![image](https://user-images.githubusercontent.com/4042181/31635041-7e2d5962-b293-11e7-919d-83219458c9dd.png)

**Comment by abbyraskinUSDS** [comment]
Waiting on final business rules from AMO.

**Comment by abbyraskinUSDS** [comment]
![image](https://user-images.githubusercontent.com/4042181/31240783-d17b852a-a9cf-11e7-867c-330fbcac11de.png)

**Comment by shanear** [comment]
@nickheiner-usds this should be all you! redux away 🚀

**Comment by nickheiner-usds** [comment]
> Step 2. Update the Subject Line with "Ramp Election."

@abbyraskinUSDS we really want to tell people to include a period in the subject line? (This applies to other steps in the copy as well.)

**Comment by abbyraskinUSDS** [comment]
No, the period should come after the quotation marks, good call. Will update AC.

**Comment by NickHeiner** [comment]
As an aside, this is why I always put the period outside the quotation marks, even in formal writing. :smile:

**Comment by abbyraskinUSDS** [comment]
Technically, (in America, at least) they're supposed to go inside...but I see the argument in this case. :)

**Comment by NickHeiner** [comment]
I know they technically are. This is my act of revolution against THE SYSTEM.

**Comment by NickHeiner** [comment]
@abbyraskinUSDS would you mind updating the AC?

**Comment by kierachell** [comment]
#### Where should `Required` text be?
<img width="674" alt="screen shot 2017-10-19 at 8 13 39 am" src="https://user-images.githubusercontent.com/18075411/31770352-7d9411b2-b4a5-11e7-81fc-3f7f2c8c3bce.png">


#### An acknowledgment with a checkbox displays below the issues on appeal. It is unselected by default. It should operate according to the styleguide.

does not exist.

**Comment by abbyraskinUSDS** [comment]
"Required" text should be closer to the end of the checkbox statement. The issues don't exist yet because they're getting built in a separate story, so the placement is fine for now. (#3583)

**Comment by kierachell** [comment]
**PASSED**

<img width="570" alt="screen shot 2017-10-20 at 9 35 58 am" src="https://user-images.githubusercontent.com/18075411/31823464-458eb7e2-b57a-11e7-8b1e-da798ca59c0f.png">
